db_test.go

```diff
@@ -25,23 +28,32 @@ type testUser struct {
 	ID                  string `json:"id,omitempty"`
 }
 
-// getOptions returns a list of options to be used when creating a new websocket connection
-func getOptions() (options []surrealdb.Option) {
-	options = append(options, surrealdb.UseWriteCompression(true), surrealdb.WithTimeout(20*time.Second))
-	return
-}
-
 func TestSurrealDBSuite(t *testing.T) {
-	// Without options
 	SurrealDBSuite := new(SurrealDBTestSuite)
-	suite.Run(t, SurrealDBSuite)
+	SurrealDBSuite.wsImplementations = make(map[string]websocket.WebSocket)
+
+	// Without options
+	SurrealDBSuite.wsImplementations["gorilla"] = gorilla.Create()
 
 	// With options
-	SurrealDBSuite.options = getOptions()
-	suite.Run(t, SurrealDBSuite)
+	SurrealDBSuite.wsImplementations["gorilla/w/opt"] = gorilla.Create().SetTimeOut(time.Minute).SetCompression(true)
```

Comment:
We may want to avoid `/` for test names as it may get mistaken for paths. I would recommend `_` instead, even if it looks ugly. It's about the tests being rerunnable conveniently.